recipes/cmweather/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
It looks like the license file is included in the pypi source distribution, so you don't have to include it here.

Reply:
@synapticarbors - I removed the license file here.